app/javascript/src/transitive_dependencies.js

```diff
@@ -1,54 +1,61 @@
-import $ from "jquery";
-
-$(document).on('click', '.deps_expanded-link', function () {
+document.addEventListener('click', function (event) {
+  const trigger = event.target.closest('.deps_expanded-link')
```

Comment:
Any reason to not use terminating `;` in this file? It seems used in other files consistently.

Reply:
Let's look to add linting to our JS code so we're consistent

Reply:
I'm happy to merge #5928 first and then lint this, since there's nothing we need to check before merging on the other PR.

Reply:
ok, all set and rebased.